scss/_variables.scss

```diff
@@ -1044,6 +1044,8 @@ $figure-caption-color:              $gray-600 !default;
 
 // Breadcrumbs
 
+$breadcrumb-font-size:              $font-size-base !default;
```

Comment:
You can use `null` for this, this will not output any css by default, but will make it possible to change the font size if needed.